wolfgang_pybullet_sim/scripts/simulation_headless.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python3
+
+from wolfgang_pybullet_sim.ros_interface import ROSInterface
+from wolfgang_pybullet_sim.simulation import Simulation
+
+if __name__ == "__main__":
+    simulation = Simulation(False)
+    ros_interace = ROSInterface(simulation)
```

Comment:
```suggestion
    ros_interface = ROSInterface(simulation)
```
Please rename to `ros_interface` here and in the other script